src/Combinatorics/Matroids/properties.jl

```diff
@@ -884,7 +884,8 @@ function tutte_connectivity(M::Matroid)
 end
 
 @doc raw"""
-    tutte_polynomial(M::Matroid)
+    tutte_polynomial(M::Matroid; parent::ZZMPolyRing)
```

Comment:
Can we indicate somehow that `parent` is optional? E.g.
```suggestion
    tutte_polynomial(M::Matroid[; parent::ZZMPolyRing])
```
but I am not sure this is universally understood. Or perhaps
```suggestion
    tutte_polynomial(M::Matroid; parent::ZZMPolyRing = ...)
```
and/or just say in the text that a parent can be *optionally* passed via kwarg or as first argument or so